### PR TITLE
ntfs-3g: use patch from upstream, disable it for head, real test

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by yurikoles** [comment]
Please merge only after https://github.com/Homebrew/formula-patches/pull/284

File: Formula/ntfs-3g.rb [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
   homepage "https://www.tuxera.com/community/open-source-ntfs-3g/"
-  url "https://tuxera.com/opensource/ntfs-3g_ntfsprogs-2017.3.23.tgz"
-  sha256 "3e5a021d7b761261836dcb305370af299793eedbded731df3d6943802e1262d5"
-  revision 1
+  revision 2
+  stable do
+    url "https://tuxera.com/opensource/ntfs-3g_ntfsprogs-2017.3.23.tgz"
+    sha256 "3e5a021d7b761261836dcb305370af299793eedbded731df3d6943802e1262d5"
+
+    # Detection of struct stat members fails Xcode 9
```
Comment (SMillerDev) [inline Formula/ntfs-3g.rb]: That's a very old unsupported version. Please make this a normal patch block

Comment (yurikoles) [reply Formula/ntfs-3g.rb]: Thanks for suggestion, this code also smells bad for me. Could you please give me example of stable-only patch?

File: Formula/ntfs-3g.rb [hunk 2]
```diff
@@ -91,7 +92,13 @@ def install
   end
 
   test do
-    output = shell_output("#{bin}/ntfs-3g --version 2>&1")
-    assert_match version.to_s, output
+    # create a small raw image, format and check it
```
Comment (SMillerDev) [inline Formula/ntfs-3g.rb:95]: 😍

**Review by SMillerDev — APPROVED** [review]

**Comment by yurikoles** [comment]
@SMillerDev ping?

**Comment by SMillerDev** [comment]
Thanks @yurikoles ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by yurikoles** [comment]
Thanks, @SMillerDev,

But how about merging it?

**Comment by SMillerDev** [comment]
It'll be merged as soon as I have time to do merges. Even in these trying times I only bring my phone when I'm having lunch 😁

**Comment by yurikoles** [comment]
Ok, many thanks, @SMillerDev. And sorry for pushing on you! :)

**Pull request closed without merging**